apps/cowswap-frontend/src/common/updaters/orders/UnfillableOrdersUpdater/types.ts

```diff
@@ -0,0 +1,5 @@
+import { Order } from 'legacy/state/orders/actions'
+
+import { ParsedOrder } from 'utils/orderUtils/parseOrder'
+
+export type OrderToCheckFillability = Order | ParsedOrder
```

Comment:
Don't we have a type where both order types are combined already? For TWAPs I think.